src/ccmain/tesseractclass.cpp

```diff
@@ -340,6 +340,9 @@ Tesseract::Tesseract()
     , BOOL_MEMBER(tessedit_create_txt, false, "Write .txt output file", this->params())
     , BOOL_MEMBER(tessedit_create_hocr, false, "Write .html hOCR output file", this->params())
     , BOOL_MEMBER(tessedit_create_alto, false, "Write .xml ALTO file", this->params())
+    , BOOL_MEMBER(tessedit_create_page, false, "Write .page.xml PAGE file", this->params())
+    , BOOL_MEMBER(tessedit_create_page_polygon, true, "Create the PAGE file with polygons instead of bbox values", this->params())
```

Comment:
```suggestion
    , BOOL_MEMBER(tessedit_create_page_polygon, true, "Create the PAGE file with polygons instead of box values", this->params())
```

Reply:
`bbox` is used internally in the Tesseract code, but not used in the documentation, so we should avoid that name.